### ISSUE TITLE
New Note Requests

### ISSUE THREAD

**Issue by xelepart** [Zamiell/hanabi-live#2248]
My play group has been longing for some new note ideas. Specifically, we're imagining a few notes that cause new icons to show up (similar to the ! critical card icon). And sometimes these feel silly to me, but we are all parents and play after the kids are asleep and we're too tired to actually be smart :D -- Here's what we're thinking:
* [play] puts a green "play" triangle icon on the card. Or maybe it gives a new highlight color (like [finesse] does).
* [save] puts a "pause" icon or a hand icon or something.
* [bluff] puts a "B" icon or does a special highlight color.

Basically, there are some notes we put on that we kinda wanna see all the time, but "sticky notes" doesn't feel right. Some small circle icons or border colors feels more in the style of the game, though?

I am terrible at icons, but I'd be happy to do the coding/pull request for this is it was approved.

**Comment by Zamiell** [comment]
any updates on this?
